src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortTermDiscount = 20;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const longTerm = 7;
+  const carRentCost = 40;
+
+  if (days < shortTerm) {
+    return days * carRentCost;
```

Comment:
Let's create a variable for `days * carRentCost` in order not to duplicate it three times